apps/convergecfd/run_ccfd_3.0_impi.pbs

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
Would it be possible to make a single script for both versions ? there is a lot of duplicated lines which can end into errors

Reply:
It's possible but I am not sure of the value for the additional work. Many things have changed between the versions and 2.4 is no longer be actively developed. I only added the 2.4 run script for those companies that have not switched over to 3.0 yet. If you would only rather to have a single run script for 3.0 then I can remove the impi extension on the filename and do some additional work when their HPCX version becomes available so that I don't need to have a run_ccfd_3.0_hpcx.pbs version